career/ladder.md

```diff
@@ -184,7 +187,7 @@ Here are some examples of other career ladders:
 - [ ] Does not take "all" responsibilities. They let their teammates take the responsibilities they care about.
 
 #### Technical skills
-- [ ] Breaks down complex projects into simple systems that can be effectively built and maintained by less experienced developers.
+- [ ] Have turned around highly complex projects so that less experienced developers could collaborate on them. They are recognized for someone who knows how to transform a legacy system into a successful project.
```

Comment:
```suggestion
- [ ] Has turned around highly complex projects so that less experienced developers could collaborate on them. They are recognized for someone who knows how to transform a legacy system into a successful project.
```